signac/contrib/utility.py

```diff
@@ -188,6 +267,29 @@ def _mkdir_p(path):
 
 
 def split_and_print_progress(iterable, num_chunks=10, write=None, desc='Progress: '):
+    """Split the progress and prints it.
+
+    Parameters
+    ----------
+    iterable : list
+        list of state points.
```

Comment:
```suggestion
    iterable : list
        list of job ids.
```
In `Project._update_memory_cache`, `iterable` is set equal to a list of job ids, rather than statepoints. In:

https://github.com/glotzerlab/signac/blob/b21bdebcd579e1056152eb76568444927904b6a3/signac/contrib/project.py#L1311-L1340
`self._sp_cache` is a dictionary, where the keys are job ids and the entries are the associated statepoint dictionaries.

On the other hand, are these descriptions meant to be specific to a specific application of the function or should they decribe the broader functionality? If they are meant to be general, it may be better to change this to something like 
```suggestion
    iterable : list
        list of values to be chunked.
```

Reply:
as far as I know a more general description of variable is fine.

Reply:
Sounds good then!